recipe/bld.bat

```diff
@@ -1,55 +1,51 @@
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%
-set LIB=%LIBRARY_LIB%;%LIB%
 
-cd qt-everywhere-opensource-src-%PKG_VERSION%
+:: TODO: OpenGL
+set "INCLUDE=%LIBRARY_INC%;%INCLUDE%"
+set "LIB=%LIBRARY_LIB%;%LIB%"
+
+:: this needs to be CALLed due to an exit statement at the end of configure:
+call configure ^
+     -prefix %LIBRARY_PREFIX% ^
+     -libdir %LIBRARY_LIB% ^
+     -bindir %LIBRARY_BIN% ^
+     -headerdir %LIBRARY_INC%\qt ^
+     -archdatadir %LIBRARY_PREFIX% ^
+     -datadir %LIBRARY_PREFIX% ^
+     -L %LIBRARY_LIB% ^
+     -I %LIBRARY_INC% ^
+     -confirm-license ^
+     -no-fontconfig ^
+     -icu ^
+     -no-separate-debug-info ^
+     -no-warnings-are-errors ^
+     -nomake examples ^
+     -nomake tests ^
+     -no-warnings-are-errors ^
+     -opengl desktop ^
+     -opensource ^
+     -openssl ^
+     -platform win32-msvc%VS_YEAR% ^
+     -release ^
+     -shared ^
+     -qt-freetype ^
+     -system-libjpeg ^
+     -system-libpng ^
+     -system-zlib ^
+     -mp
```

Comment:
It passes the [/mp](https://msdn.microsoft.com/en-us/library/bb385193.aspx) option to `cl`. I found building with `jom` just randomly failed. Building without `jom` but with `/mp` seems to keep the build time down to about 1 hour.